### DISCUSSION TITLE
Straw mobile running rustmas

### DISCUSSION THREAD

**Discussion by 2m** [mrozycki/rustmas#290] [Show and tell]
I discovered this project last year and successfully "deployed" rustmas to my Christmas tree. For this holiday season aspirations were higher and my friend suggested to build a traditional Lithuanian straw mobile out of 1500 LED lights (~26 meters of LED strip).

I am very glad how it turned out. Thanks again for working on rustmas and sharing it with the community ❤️ 

Here is a short blog post of the process of building it: https://2m.lt/blog/straw-mobile-and-eulerian-circuits

![FB_IMG_1734370397277](https://github.com/user-attachments/assets/f8cf45f7-c388-4e92-bd83-f5b797fdc2db)
![FB_IMG_1734370400592](https://github.com/user-attachments/assets/9a7035b3-7b30-4ae5-aec6-f76424ccac18)
![FB_IMG_1734370407261](https://github.com/user-attachments/assets/38054838-e0f3-43b4-a35d-c413ba48c4f2)

https://github.com/user-attachments/assets/07b506a6-3c46-4dd0-b98a-5ac59c4f4696

**Comment by mrozycki** [comment]
Wow, looks amazing!

I'm wondering how the performance is with 1500 LEDs, I'm assuming on a single pico. We calculated a while back that to get 30FPS you can't have much more than 1000 lights, but in the video it actually looks quite smooth.

Have you written any of your own animations? 

Thanks a lot for sharing!

**Reply by 2m** [reply]
Yes, it runs on a single Pico W. We also used wireless connectivity and surprisingly no performance problems.

No custom animations yet. But I am planning to give it a go soon. :)
